docs/.vuepress/handsontable-manager/dependencies.js

```diff
@@ -32,7 +32,7 @@ const getFixer = () => {
  * The function `buildDependencyGetter` is the best place to care about that.
  *
  * @param {string} version The current selected documentation version.
- * @returns {function({dependency:string}):[string,string[],string]} [jsUrl, dependentVars[]?, cssUrl?].
```

Comment:
Why this was changes? AFAIK it was correct

Reply:
The syntax is not currently supported by the `eslint-plugin-jsdoc` plugin.
![image](https://user-images.githubusercontent.com/571316/121888203-ff885a00-cd17-11eb-91d2-c2cc33fa43ee.png)

Reply:
Ok, so please keep `[jsUrl, dependentVars[]?, cssUrl?]` description for this param if you could 😄

Reply:
Done 6fb6a0948526a33dc21fa9d8c1323973d45bcca6.